src/screens/ProfileScreen/DetailScreen/ActiveTaskDetail.tsx

```diff
@@ -3,64 +3,41 @@ import { StyleSheet } from 'react-native';
 import { View, Text, TouchableOpacity, ScrollView } from 'react-native';
 import { useNavigation, useRoute } from '@react-navigation/native';
 import ProgressModal from '../../../components/Modal/ProgressModal';
-// import { getDateAndTimeFromUnix } from '../../AuthScreen/Util';
 
 const ActiveTaskDetail = () => {
   const route = useRoute();
-  const { task } = route.params;
+  const { task } = route.params as any; // TODO fix types when used via route navigation
```

Comment:
Can't we define a type for this?

Reply:
No as it is not being used via route navigation, so no data being come in task variable, so when someone uses this, they can define the types at that time. That's why I wrote TODO and put it as any for now